src/Page/Explore/TextileMaterials.elm

```diff
@@ -69,11 +72,21 @@ table db { detailed, scope } =
                         [ Origin.toMicrofibersComplement origin
                             |> Unit.impactToFloat
                             |> Format.formatImpactFloat { unit = "\u{202F}Pts/kg", decimals = 2 }
+                        , Link.smallPillExternal
+                            [ href (Gitbook.publicUrlFromPath Gitbook.TextileComplementMicrofibers) ]
+                            [ Icon.question ]
                         ]
           }
         , { label = "Procédé de fabrication du fil"
           , toValue = Table.StringValue <| .origin >> Origin.threadProcess
-          , toCell = .origin >> Origin.threadProcess >> text
+          , toCell =
+                \{ origin } ->
+                    div [ classList [ ( "text-center", not detailed ) ] ]
+                        [ Origin.threadProcess origin |> text
+                        , Link.smallPillExternal
+                            [ href (Gitbook.publicUrlFromPath Gitbook.TextileSpinning) ]
+                            [ Icon.question ]
+                        ]
```

Comment:
We are duplicating this code 3 times, I think it should maybe moved into a function?

Reply:
You're right, I've refactored the four similar patterns into a function, used in a point-free style